### PR TITLE
Task/3 prestashop init backup

### PR DESCRIPTION
**Pull request by fildaw**
Implemented backuping and restoring of prestashop database. Added base of prestashop for later configuration throughout the task.

Partially implemented #3

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -45,3 +48,21 @@ If you want to change the domain the shop is available at, you have to update it
 
 > [!CAUTION]
 > This method is not ready - we need a method to save the database and the files in the repository.
```
Comment (Nasus20202) [inline src/README.md]: Can we remove this disclaimer - the issue seems to be solved?

File: src/db_backup/Dockerfile
Comment (Nasus20202) [inline src/db_backup/Dockerfile:1]: This file seems to be unused :)

**Review by Nasus20202 — APPROVED** [review]
Good job 😄

File: src/README.md (hunk 1 above)
Comment (fildaw) [reply src/README.md]: Sure, will remove it

**Review by KanarekLife — APPROVED** [review]
lgtm :)

**Pull request merged**